src/theme/components/link.ts

```diff
@@ -8,211 +10,233 @@ import {
 } from "./button";
 import { screenreaderOnly } from "./globalMixins";
 
-export const baseLinkStyles = {
-  color: "ui.link.primary",
-  textDecoration: "underline",
-  textDecorationStyle: "dotted",
-  textDecorationThickness: "1px",
-  textUnderlineOffset: "2px",
-  _dark: {
-    color: "dark.ui.link.primary",
-  },
-  _hover: {
-    color: "ui.link.secondary",
+const { defineMultiStyleConfig, definePartsStyle } =
+  createMultiStyleConfigHelpers(["base", "screenreaderOnly"]);
```

Comment:
Add the comments for this function as you did in `button.ts`.

Reply:
Can do this when the `Link` component goes through a better update.